src/bindings/jna/libelektra/dependencies.gradle

```diff
@@ -5,4 +5,5 @@ dependencies {
 
 	// test dependencies
 	testImplementation "junit:junit:$junitVersion"
+	testImplementation "org.mockito:mockito-core:$mockitoVersion"
```

Comment:
@tucek any thoughts about which test framework we want to use?

Reply:
mockito is usually a good choice in my opinion